index.html

```diff
@@ -18,22 +18,128 @@
+        <img src="./assets/Snapshoot Portfolio (2).png" alt="Snapshoot-1" />
+        <h2 class="project-title">Tonic</h2>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (3).png" alt="Snapshoot-1" />
+        <h2 class="project-title">Multi-Post Stories</h2>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="about-section">
+        <h2 class="about-me-text">About Myself</h2>
+        <p class="about-me-description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="social-media">LET’S CONNECT</p>
+        <ul class="icons">
+          <li><img src="./assets/Vector (1).svg" alt="github" /></li>
+          <li><img src="./assets/Linkedin icon.svg" alt="linkedin" /></li>
+          <li><img src="./assets/Vector (3).svg" alt="s" /></li>
+          <li><img src="./assets/Vector.svg" alt="twitter" /></li>
+          <li><img src="./assets/Vector (2).svg" alt="m" /></li>
+        </ul>
+        <button class="btn get-my-resume">Get my resume</button>
+        <ul class="skills">
+          <li class="skill-name-and-arrow">
+            Languages
+            <span><img src="./assets/down-arrow.png" alt="down-arrow" /></span>
+          </li>
+          <li><img src="./assets/Frame 9.png" alt="js" />
+          <img src="./assets/Frame 10.png" alt="html" />
+          <img src="./assets/Frame 11.png" alt="css" /></li>
+          <li class="skill-name-and-arrow">Frameworks<span><img src="./assets/arrow-point-to-right.png" alt="rigth-arrow" /></li>
+          <hr />
+          <li class="skill-name-and-arrow">Skills<span><img src="./assets/arrow-point-to-right.png" alt="right-arrow" /></li>
```

Comment:
- [ ] [OPTIONAL] You have reached a great milestone. Congratulations :clap: . However i would suggest to fix the `<hr>` underlines below `li` to have the same color as the one in figma. Consider the images below for clearance :muscle: 
- Your design :sunglasses: 
![Screenshot from 2023-01-14 22-21-38](https://user-images.githubusercontent.com/99394374/212492611-b793cdc6-8ebe-488c-831d-8c322df455eb.png)

- Figma design :sunglasses: 
![Screenshot from 2023-01-14 22-21-52](https://user-images.githubusercontent.com/99394374/212492625-de3843d7-7d14-4e4d-aebc-113f246b7e7f.png)